doc/source/python_reference.rst

```diff
@@ -579,6 +579,8 @@ Functions:
    apply_inverse_epochs
    apply_inverse_raw
    compute_rank_inverse
+   psd_epochs_source
+   psd_raw_source
```

Comment:
...and again, we have `apply_inverse_epochs` and `apply_inverse_raw` but then to match the changes to the TF module we need `psd_epochs_source` and `psd_epochs_raw`. It seems like the proper match here is what we already had, which was `compute_psd_raw` and `compute_psd_epochs`. So I wonder if the *other* TF functions are actually what should be renamed.